frontend/src/modules/organization/components/view/organization-view-header.vue

```diff
@@ -229,18 +187,75 @@ const formattedInformation = (value, type) => {
   // Render inforamation depending on type
   if (type === 'date') {
     return formatDate({ timestamp: value });
-  } if (type === 'number') {
+  }
+  if (type === 'number') {
     return formatNumber(value);
-  } if (type === 'relative') {
+  }
+  if (type === 'relative') {
     return formatDateToTimeAgo(value);
```

Comment:
For this ticket all you had to do is just check if date is 1. january 1970 and if so display 'some time ago' in function formatDateToTimeAgo no else logic needed

Reply:
the `formatDateToTimeAgo`  function already have the `some time ago` implemented